<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1103</width>
    <height>663</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>URNA CEIT</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>821</width>
      <height>621</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>1000</height>
     </size>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tela1">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>200</y>
        <width>541</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 74, 173);</string>
      </property>
      <property name="text">
       <string>Insira seu Token de aluno:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>320</y>
        <width>541</width>
        <height>60</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	background: #b2c8e6;
	border: 0 px solid #004AAD;
	color: rgb(0, 29, 69);
	border-radius: 10px;
}

QLineEdit:focus {
	border: 2px solid #FBAD25;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>12</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_login">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>400</y>
        <width>231</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(0, 74, 173);
	color: rgb(255, 255, 255);
	border-radius: 10px;
}

QPushButton:hover {
	background-color: rgb(25, 92, 181);
}</string>
      </property>
      <property name="text">
       <string>Próximo</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tela2">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>621</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 74, 173);</string>
      </property>
      <property name="text">
       <string>Escolha a chapa em que deseja votar:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_chapa">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>440</y>
        <width>231</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(0, 74, 173);
	color: rgb(255, 255, 255);
	border-radius: 10px;
}

QPushButton:hover {
	background-color: rgb(25, 92, 181);
}</string>
      </property>
      <property name="text">
       <string>Próximo</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>210</y>
        <width>450</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 74, 173);</string>
      </property>
      <property name="text">
       <string>Chapa 2</string>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_1">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>140</y>
        <width>450</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 74, 173);</string>
      </property>
      <property name="text">
       <string>Chapa 1</string>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>280</y>
        <width>450</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 74, 173);</string>
      </property>
      <property name="text">
       <string>Chapa 3</string>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>350</y>
        <width>450</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 74, 173);</string>
      </property>
      <property name="text">
       <string>Branco</string>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tela3">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>230</y>
        <width>160</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 74, 173);</string>
      </property>
      <property name="text">
       <string>Tesoureiro Geral:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>130</y>
        <width>160</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 74, 173);</string>
      </property>
      <property name="text">
       <string>Presidente: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>280</y>
        <width>160</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 74, 173);</string>
      </property>
      <property name="text">
       <string>Diretor Pedagógico:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>230</y>
        <width>160</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 74, 173);</string>
      </property>
      <property name="text">
       <string>1º Secretário</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>380</y>
        <width>160</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 74, 173);</string>
      </property>
      <property name="text">
       <string>Diretor de Esporte:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>380</y>
        <width>160</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 74, 173);</string>
      </property>
      <property name="text">
       <string>Diretor Social:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_cancelar">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>490</y>
        <width>231</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(0, 74, 173);
	color: rgb(255, 255, 255);
	border-radius: 10px;
}

QPushButton:hover {
	background-color: rgb(25, 92, 181);
}</string>
      </property>
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>280</y>
        <width>160</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 74, 173);</string>
      </property>
      <property name="text">
       <string>1º Tesoureiro:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>430</y>
        <width>160</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 74, 173);</string>
      </property>
      <property name="text">
       <string>1º Suplente: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>330</y>
        <width>160</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 74, 173);</string>
      </property>
      <property name="text">
       <string>Diretor de Imprensa: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>541</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 74, 173);</string>
      </property>
      <property name="text">
       <string>Deseja Confirmar Seu Voto?</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>430</y>
        <width>160</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 74, 173);</string>
      </property>
      <property name="text">
       <string>2º Suplente: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>180</y>
        <width>160</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 74, 173);</string>
      </property>
      <property name="text">
       <string>Secretário Geral: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>180</y>
        <width>160</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 74, 173);</string>
      </property>
      <property name="text">
       <string>Vice-Presidente: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>330</y>
        <width>160</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 74, 173);</string>
      </property>
      <property name="text">
       <string>Diretor de Cultura: </string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_votar">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>490</y>
        <width>231</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(0, 74, 173);
	color: rgb(255, 255, 255);
	border-radius: 10px;
}

QPushButton:hover {
	background-color: rgb(25, 92, 181);
}</string>
      </property>
      <property name="text">
       <string>Votar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tela4">
     <widget class="QPushButton" name="btn_finalizar">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>330</y>
        <width>231</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(0, 74, 173);
	color: rgb(255, 255, 255);
	border-radius: 10px;
}

QPushButton:hover {
	background-color: rgb(25, 92, 181);
}</string>
      </property>
      <property name="text">
       <string>Finalizar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>160</y>
        <width>541</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 74, 173);</string>
      </property>
      <property name="text">
       <string>SEU VOTO FOI REGISTRADO COM SUCESSO!</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_34">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>260</y>
        <width>371</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 74, 173);</string>
      </property>
      <property name="text">
       <string>Clique em Finalizar.</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tela5">
     <widget class="QPushButton" name="btn_fechar_resultados">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>330</y>
        <width>231</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(0, 74, 173);
	color: rgb(255, 255, 255);
	border-radius: 10px;
}

QPushButton:hover {
	background-color: rgb(25, 92, 181);
}</string>
      </property>
      <property name="text">
       <string>Finalizar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>160</y>
        <width>541</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 74, 173);</string>
      </property>
      <property name="text">
       <string>SEU VOTO FOI REGISTRADO COM SUCESSO!</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_34">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>260</y>
        <width>371</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 74, 173);</string>
      </property>
      <property name="text">
       <string>Clique em Finalizar.</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="barra_topo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 74, 173);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="usuario">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>-10</y>
       <width>400</width>
       <height>60</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Usuário: Nenhum</string>
     </property>
    </widget>
    <widget class="QLabel" name="titulo">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>-10</y>
       <width>541</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>ELEIÇÕES GRÊMIO ESTUDANTIL CEIT - 2022</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>225</y>
      <width>320</width>
      <height>320</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Logo CEIT.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
